packages/oauth/src/stores/index.ts

```diff
@@ -0,0 +1,2 @@
+export { MemoryInstallationStore } from './memory-store';
```

Comment:
Both exported here to clean up the import, ie:

`import { MemoryInstallationStore, FileInstallationStore } from './stores';`

Reply:
hm, do we need this file? Just importing the two in `src/index.ts` sounds good enough to me.